Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.ComponentModel.DataAnnotations;
```

Comment:
Yep, I used it for autofixture [here](https://github.com/AutoFixture/AutoFixture/compare/v4...adamchester:netcore-step2#diff-491c68353068d4c77e3a8a02d1a84ee6R41)